libthymesisflow/cmd/thymesisf-agent.c

```diff
@@ -0,0 +1,150 @@
+#include "proto.h"
```

Comment:
There is no copyright notice in any of the source files, please add to all files missing it (both .h and .c).